theseus_gui/src/pages/Index.vue

```diff
@@ -0,0 +1,32 @@
+<script setup>
+import { useInstances, useNews } from '@/store/state'
+import RowDisplay from '@/components/RowDisplay.vue'
+
+const instances = useInstances()
+const news = useNews()
+instances.fetchInstances()
+news.fetchNews()
+
+const recentInstances = instances.instances.slice(0, 4)
+const popularInstances = instances.instances.filter((i) => i.downloads > 50)
+const trendingMods = instances.instances.filter((i) => i.trending)
+</script>
+
+<template>
+  <div class="page-container">
+    <RowDisplay label="Jump back in" :instances="recentInstances" :canPaginate="false" />
+    <RowDisplay label="Popular packs" :instances="popularInstances" :canPaginate="true" />
+    <RowDisplay label="News & updates" :news="news.news" :canPaginate="true" />
+    <RowDisplay label="Trending mods" :instances="trendingMods" :canPaginate="true" />
```

Comment:
The text labels here should be the pure white/black and the lines should be a bit darker. The pagination toggles (if they don't already) should have a background. Also make sure each of these works with the scroll wheel (so you can use it alongside the buttons)

Reply:
text should also be a bit bigger

Reply:
I was able to get some code up to detect the scrolling event inside the respective rows, but we're going to have an issue detecting whether the user is scrolling up or down. It seems the usual solution is to store the window's Y offset and compare as you scroll, but that won't work for us scrolling content horizontally in a div.

I've made the styling changes you mentioned but am omitting the scrolling for now until we come up with something.